backend/services/esweService.ts

```diff
@@ -55,10 +57,42 @@ const compareByDate = (dateA: Moment, dateB: Moment, descending = true) => {
   return 0
 }
 
+const renameAssessmentLabel = (qualificationType) =>
+  qualificationType === AssessmentQualificationType.English ? 'English/Welsh' : qualificationType
+
+const createSkillAssessmentSummary = (learnerAssessment: curious.LearnerAssessment) => {
+  const { qualification, establishmentName } = learnerAssessment || {}
+  const { qualificationType, qualificationGrade, assessmentDate } = qualification || {}
+
+  if (!assessmentDate) {
+    return [
+      {
+        label: renameAssessmentLabel(qualificationType),
+        value: AWAITING_ASSESSMENT_CONTENT,
+      },
+    ]
+  }
+
+  return [
+    {
+      label: renameAssessmentLabel(qualificationType),
+      value: qualificationGrade,
+    },
+    {
+      label: 'Assessment date',
+      value: readableDateFormat(assessmentDate, CURIOUS_DATE_FORMAT),
+    },
+    {
+      label: 'Assessment location',
+      value: establishmentName,
+    },
+  ]
+}
+
 /**
  * Education skills and work experience (ESWE)
  */
-export default class EsweService {
+export class EsweService {
```

Comment:
**Not blocking**
Was there a reason this was changed from a default export?

Reply:
I was under the impression if you had a default export you couldn't export other things separately - not sure why, I'll swap it back